Add new blog entry: What is Terraform Cloud and why you might need it? (#52)
xNok/xNok: website/content/english/blogs/2022-03-04-what-is-terraform-cloud-and-why-you-might-need-it.md

---
title: "💡 What is Terraform Cloud and why you might need it?"
date: 2022-03-04
draft: false
categories: ["DevOps", "Terraform"]
tags: ["Terraform", "Terraform Cloud", "IaC"]
images:
- https://miro.medium.com/v2/resize:fit:720/format:webp/1*KkKtK1D5ZVmJGYqj9Yeq2w.png
---

> Original Article: [What is Terraform Cloud and why you might need it?](https://faun.pub/what-is-terraform-cloud-and-why-you-might-need-it-c9847fb8f6e6)

## Summary

This article explains **Terraform Cloud**, a platform that helps you manage and automate your **infrastructure as code (IaC)**. You'll discover its key features, including **collaboration**, **automation**, and **consistency** across cloud environments. The article emphasizes the benefits of using Terraform Cloud to streamline infrastructure management and improve team workflows. It also touches on how Terraform Cloud enables you to maintain version control, track changes, and enforce policies for your infrastructure.

## Key Concepts

> *   **Terraform Cloud**: A HashiCorp platform for managing and automating infrastructure as code.
> *   **Collaboration**: Facilitates team collaboration on infrastructure projects with features like access control and shared state.
> *   **Automation**: Automates infrastructure provisioning and management tasks, reducing manual effort and errors.
> *   **Consistency**: Ensures consistency across cloud environments by enforcing policies and configurations.

Author: ```suggestion
*   **Terraform Cloud**: A Terraform based HashiCorp platform for managing and automating infrastructure as code.
```

> 

Author: ``` suggestion
## Update

I have since moved away from Terraform Cloud. While the workflow they offer is rather good in practice, it was too much more practical to have Ansible run my Terraform workflows since the end goal is to provision the VM created in the process.
```
